tests/Block/AdminSearchBlockServiceTest.php

```diff
@@ -35,12 +36,19 @@ class AdminSearchBlockServiceTest extends BlockServiceTestCase
      */
     private $searchHandler;
 
+    /**
+     * @var TemplateRegistryInterface
+     */
+    private $templateRegistry;
+
     protected function setUp(): void
     {
         parent::setUp();
 
         $this->pool = $this->createMock(Pool::class);
         $this->searchHandler = $this->createMock(SearchHandler::class);
+        $this->templateRegistry = $this->prophesize(TemplateRegistryInterface::class);
+        $this->templateRegistry->getTemplate('search_result_block')->willReturn('@SonataAdmin/Block/block_search_result.html.twig');
```

Comment:
If needed, this will be the syntax

```suggestion
        $this->templateRegistry->method('getTemplate')->willReturn('@SonataAdmin/Block/block_search_result.html.twig');
```